agent/src/trident.rs

```diff
@@ -1075,53 +1075,6 @@ impl DomainNameListener {
     }
 }
 
-// pub enum Components {
```

Comment:
It is recommended to commit in the form of an amend so that no multiple commits are included in the same pr.